src/styles/variables.scss

```diff
@@ -0,0 +1,4 @@
+$acsentColor: #00acdc;
```

Comment:
There is a typo in the variable name `$acsentColor`. It should be `$accentColor` to correctly reflect the term 'accent'.